src/network/connectionpool.py

```diff
@@ -67,6 +66,7 @@ def isAlreadyConnected(self, nodeid):
     def addConnection(self, connection):
         if isinstance(connection, UDPSocket):
             return
+        logger.warning('addConnection %s -> %s', connection, connection.destination)
```

Comment:
I would use `logger.debug`.

Reply:
All warnings are in 42e32a3 which will be dropped before merging the PR.